Plugins/SafeDIGenerateDependencyTree/SafeDIGenerateDependencyTree.swift

```diff
@@ -112,3 +133,17 @@ extension SafeDIGenerateDependencyTree: XcodeBuildToolPlugin {
     }
 }
 #endif
+
+extension Data {
+    fileprivate func write(toPath filePath: String) throws {
```

Comment:
This extension also exists in another module, but this plug in module cannot depend on any other library we have in this package